### DISCUSSION TITLE
The default spacing between inline-block elements is gone after rsbuild rendering.

### DISCUSSION THREAD

**Discussion by shiwenna** [web-infra-dev/rsbuild#4204] [Q&A]
![image](https://github.com/user-attachments/assets/f7659a94-1b40-4eaf-8a2d-aa8aca861648)
![image](https://github.com/user-attachments/assets/ec85461a-ee90-4ec7-b95a-c1829b2ce671)

![image](https://github.com/user-attachments/assets/d2ea4524-5472-4d0b-92c3-cf37e411b075)

**Comment by grossbart** [comment]
I've seen a similar problem when migrating from another app. Turns out the other app added a `<!DOCTYPE html>` automagically, but with Rsbuild I needed to add it myself (which makes total sense and is the correct way to do this).

So maybe it's that and your app is running in [quirks mode](https://en.wikipedia.org/wiki/Quirks_mode)?
